UI/src/view/assetFormFactory.js

```diff
@@ -488,9 +488,10 @@
     var me = this;
     var _assetTypeConfiguration;
 
-    me.initialize = function(assetTypeConfiguration){
+    me.initialize = function(assetTypeConfiguration, feedbackModel){
       var rootElement = $('#feature-attributes');
       _assetTypeConfiguration = assetTypeConfiguration;
+      new FeedbackDataTool().initialize(feedbackModel, assetTypeConfiguration.layerName, assetTypeConfiguration.authorizationPolicy, assetTypeConfiguration.singleElementEventCategory);
```

Comment:
could be replace by
`new FeedbackDataTool(feedbackModel, assetTypeConfiguration.layerName, assetTypeConfiguration.authorizationPolicy, assetTypeConfiguration.singleElementEventCategory);`